README.md

```diff
@@ -77,22 +75,30 @@ According to the template files in the templates directory, new files will be ge
 npx vue-sfc-cli -u
 ```
 
-`—files`
+`—-files`
 
 If you want to update additional files, you can pass this option, multiple files use `,` to separate
 
 ```
 npx vue-sfc-cli -u --files package.json,.babelrc.js
 ```
 
-`—test`
+`-—test`
 
 Generate a component template for testing, commonly used in CI .
 
 ```
 npx vue-sfc-cli --test
 ```
 
+`--name`, `--owner`
+
+This pair of arguments is usually used together. generate component by command to skipped questioning.
```

Comment:
these two arguments are usually used together to generate component without questioning.